clappr/dash-shaka-playback
# Adding load method for playback pattern creation

## Summary

1. This PR adds a Load method, implementing a pattern for Clappr playbacks
2. Update outdated dependencies

## How to test

1. Run the project
2. Create a Window Player object, adding this lines on `public/index.html` file, at `line 45`:
```javascript
window.player = player
```
3. Play the video
4. Load another media and play it using:
```javascript
player.core.activePlayback.load('http://dash.edgesuite.net/dash264/TestCases/2c/qualcomm/2/MultiRes.mpd')
player.core.activePlayback.play()
```
6. The video should be played correctly.
